tests/core_tests/chaingen.cpp

```diff
@@ -567,6 +567,21 @@ bool construct_tx_to_key(const std::vector<test_event_entry>& events, cryptonote
   return construct_tx(from.get_keys(), sources, destinations, from.get_keys().m_account_address, std::vector<uint8_t>(), tx, 0);
 }
 
+
+bool construct_stake_tx_to_key(const std::vector<test_event_entry> &events, transaction &tx, const block &blk_head, const account_base &from, const account_base &to,
+                               uint64_t amount, uint64_t fee, size_t nmix,
+                               const public_key &supernode_id, const signature &supernode_sign, uint64_t unlock_height)
```

Comment:
i think unlock_height should be unlock_time, which is what is used everywhere else